nixos/modules/config/shells-environment.nix

```diff
@@ -108,14 +108,18 @@ in
     };
 
     environment.shellAliases = mkOption {
-      default = {};
+      default = mapAttrs (name: mkDefault) {
+        ls = "ls --color=tty";
+        ll = "ls -l";
```

Comment:
I don't think this should work like that. It needs to be in the config section. How it's currently will disable all default aliases if a user adds any of their own.

Reply:
Oh, right. Sorry.